### PR TITLE
[CP] 4.3 GA and cherrypicks

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by sjha4** [comment]
Not to be a pain, but can we get this https://projects.theforeman.org/issues/34147 cherry-picked as well.. 🥺

**Comment by chris1984** [comment]
> Not to be a pain, but can we get this https://projects.theforeman.org/issues/34147 cherry-picked as well.. 🥺

sure :)

**Comment by jeremylenz** [comment]
[test katello]

File: lib/katello/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "4.3.0.rc4".freeze
+  VERSION = "4.3.0".freeze
```
Comment (jeremylenz) [inline lib/katello/version.rb:2]: You may want to rebase so that this commit is last, that way the tag will be on the version bump commit.

**Comment by evgeni** [comment]
[test katello]

**Comment by jeremylenz** [comment]
[test katello]

**Comment by jeremylenz** [comment]
[test katello]

**Comment by evgeni** [comment]
I think the issue is with some indirect npm dependency 🤷‍♀️

**Comment by jeremylenz** [comment]
yeah there's 2 intermittent failures we're dealing with. The unexpected end of JSON input during npm install was yours, the most recent was this other one: https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/9699/pipeline/
https://projects.theforeman.org/issues/33689

**Comment by chris1984** [comment]
[test katello]

**Comment by chris1984** [comment]
[test katello]

**Comment by chris1984** [comment]
[test katello]

**Comment by chris1984** [comment]
blah it still keeps failing

**Comment by jeremylenz** [comment]
```

[2022-01-04T15:03:47.473Z] Error:

[2022-01-04T15:03:47.473Z] Katello::KTEnvironment::main#test_0002_anonymous:

[2022-01-04T15:03:47.473Z] Psych::DisallowedClass: Tried to load unspecified class: Time

[2022-01-04T15:03:47.473Z]     test/test_helper.rb:67:in `before_setup'
```
not sure what's going on here.. @jlsherrill ring any bells?

**Comment by jeremylenz** [comment]
the above failure appears to be different from the previous two failures. Progress!

**Comment by evgeni** [comment]
🍏

Finally ;)

Not sure you want to wait for the issues I mentioned on IRC earlier, or follow up with a .1 soon, up to you.

**Comment by chris1984** [comment]
@jeremylenz moved my commit to the top, can i get an ack

**Comment by chris1984** [comment]
> 🍏
> 
> Finally ;)
> 
> Not sure you want to wait for the issues I mentioned on IRC earlier, or follow up with a .1 soon, up to you.

@evgeni I will do a .1 shortly, I wanted to get this one out since it has been on going before the shutdown.

**Review by jeremylenz — APPROVED** [review]
APJ :+1:

**Pull request merged**